[stance1:train] Activating env....
Activating env [py38ptpl]
guild_version:             0.8.0
guild_install_location:    /home/serrano/miniconda3/envs/py38ptpl/lib/python3.8/site-packages/guild
guild_home:                /home/serrano/projects/SMM4H22/tracking/.guild
guild_resource_cache:      /home/serrano/projects/SMM4H22/tracking/.guild/cache/resources
installed_plugins:         config_flags, cpu, dask, disk, dvc, exec_script, gpu, ipynb, keras, memory, perf, python_script, queue, skopt
python_version:            3.8.13 (default, Mar 28 2022, 11:38:47) [GCC 7.5.0]
python_exe:                /home/serrano/miniconda3/envs/py38ptpl/bin/python
platform:                  Linux 4.18.0-193.el8.x86_64 x86_64
psutil_version:            5.8.0
tensorboard_version:       2.6.0
cuda_version:              11.4
nvidia_smi_version:        470.82.01
latest_guild_version:      0.8.1
A newer version of Guild AI is available. Run 'pip install guildai --upgrade' to install it.
*** Check tha [guild_home] is correctly set ***
===============================================
*** Available operations (execute _guild help_ for more info) ***
Refreshing flags...
WARNING: cannot import flags from src/stance_preprocess.py: ModuleNotFoundError: No module named 'fasttext' (run with guild --debug for details)
WARNING: cannot import flags from src/prepare_txts.py: ModuleNotFoundError: No module named 'negspacy' (run with guild --debug for details)
prepare_data                      Generate Dataframes with hydrated examples from previously processed splits, texts, annotations
prepare_splits                    Create a summary table with reference to all files and assign a split to each entry
prepare_submission                from predictions in DF (subtask 2 & 3 ) generate ann files for submission
prepare_task2_stance              Prepare official training data for stance Task2
prepare_texts                     Preprocess txt files (clean, segment, etc.)
base-model:evaluate               Run official evalation script
stance1:inference                 apply transforemr
stance1:inference-sentiment       apply transforemr
stance1:pipeline_inference_event  
stance1:train                     Train model Transformer based classifier
Running script ....
New Run
Refreshing flags...
WARNING: cannot import flags from src/stance_preprocess.py: ModuleNotFoundError: No module named 'fasttext' (run with guild --debug for details)
WARNING: cannot import flags from src/prepare_txts.py: ModuleNotFoundError: No module named 'negspacy' (run with guild --debug for details)
Resolving prepare_task2_stance dependency
Using run 6d4c5793a53c4ff2bf830020a0248f0f for prepare_task2_stance resource
INFO: [numexpr.utils] Note: NumExpr detected 20 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
INFO: [numexpr.utils] NumExpr defaulting to 8 threads.
INFO: [pytorch_lightning.utilities.seed] Global seed set to 42
Reading dataset.
Traceback (most recent call last):
  File "/home/serrano/projects/SMM4H22/tracking/.guild/runs/53a5960e82c84e8480a86b31f1bbf5cd/.guild/sourcecode/src/seqcls_transformer_train.py", line 202, in <module>
    main(args)
  File "/home/serrano/projects/SMM4H22/tracking/.guild/runs/53a5960e82c84e8480a86b31f1bbf5cd/.guild/sourcecode/src/seqcls_transformer_train.py", line 63, in main
    data = Data(**dict_args)
  File "/home/serrano/projects/SMM4H22/tracking/.guild/runs/53a5960e82c84e8480a86b31f1bbf5cd/.guild/sourcecode/src/plibs/estimators/stance_datamodule.py", line 115, in __init__
    _train_data = self.read_data(train=self.hparams.data_split_train, val=None, test=None)['train'] # preload for checking size and labels
  File "/home/serrano/projects/SMM4H22/tracking/.guild/runs/53a5960e82c84e8480a86b31f1bbf5cd/.guild/sourcecode/src/plibs/estimators/stance_datamodule.py", line 176, in read_data
    dataset = datasets.DatasetDict({ k : datasets.Dataset.from_pandas( self.read_dataset(data_files[k], sample_size= (self.subsample if 'train' in k and self.subsample != 1.0 else sample_size)) ) for k in data_files }) # load as pandas dataframe
  File "/home/serrano/projects/SMM4H22/tracking/.guild/runs/53a5960e82c84e8480a86b31f1bbf5cd/.guild/sourcecode/src/plibs/estimators/stance_datamodule.py", line 176, in <dictcomp>
    dataset = datasets.DatasetDict({ k : datasets.Dataset.from_pandas( self.read_dataset(data_files[k], sample_size= (self.subsample if 'train' in k and self.subsample != 1.0 else sample_size)) ) for k in data_files }) # load as pandas dataframe
  File "/home/serrano/projects/SMM4H22/tracking/.guild/runs/53a5960e82c84e8480a86b31f1bbf5cd/.guild/sourcecode/src/plibs/estimators/stance_datamodule.py", line 197, in read_dataset
    df = df.sample(n=sample_size) if sample_size > 0 else df # now there are columns with Nones
  File "/home/serrano/miniconda3/envs/py38ptpl/lib/python3.8/site-packages/pandas/core/generic.py", line 5446, in sample
    size = sample.process_sampling_size(n, frac, replace)
  File "/home/serrano/miniconda3/envs/py38ptpl/lib/python3.8/site-packages/pandas/core/sample.py", line 99, in process_sampling_size
    raise ValueError("Only integers accepted as `n` values")
ValueError: Only integers accepted as `n` values
---------- FINALIZED -------------
